workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/confidential/AnonymisePartiesFlow.kt

```diff
@@ -20,7 +20,9 @@ class AnonymisePartiesFlow(
     @Suspendable
     override fun call(): Map<Party, AnonymousParty> {
         val sessionParties = sessions.map(FlowSession::counterparty)
-        require(sessionParties.containsAll(parties)) { "You must provide sessions for all parties." }
+        val partiesWithoutASession = parties.minus(sessionParties)
```

Comment:
I think that types don't match up? parties are list of Party, sessionParties are FlowSessions

Reply:
it maps to the FlowSession.counterparty when it's declared on line 22

Reply:
true, I haven't woken up :D